src/__tests__/ParseQuery-test.js

```diff
@@ -130,6 +140,20 @@ describe('ParseQuery', () => {
         }
       }
     });
+
+    const size = 'medium';
+    const stock = true;
+    q.notEqualTo({ size, stock });
+    expect(q.toJSON()).toEqual({
```

Comment:
Can you add an integration test?

Reply:
Where do I find an integrative test in this file?

Reply:
`/integration/test/ParseQueryTest.js`

Reply:
Done.. ;)